api/serializers/canteen.py

```diff
@@ -325,6 +325,9 @@ def get_is_managed_by_user(self, obj):
 # remember to update TD version if you update this
 class CanteenTeledeclarationSerializer(serializers.ModelSerializer):
     sectors = SectorSerializer(many=True, read_only=True)
+    central_producer_siret = serializers.SerializerMethodField(read_only=True)
+    satellite_canteens_count = serializers.SerializerMethodField(read_only=True)
+    line_ministry = serializers.SerializerMethodField(read_only=True)
```

Comment:
also daily meal count for CENTRAL

Reply:
Bien vu : https://github.com/betagouv/ma-cantine/pull/2320/commits/52b73ec0bd45e52926f54b83bbd7684bfcccf304